cmake/templates/_setup_util.py.in

```diff
@@ -3,7 +3,7 @@
 
 # Software License Agreement (BSD License)
 #
-# Copyright (c) 2012, Willow Garage, Inc.
+# Copyright (c) 2012, 2013, 2014, 2015, 2016 Willow Garage, Inc.
```

Comment:
Willow hasn't been copyrighting anything for the last few years, but this should probably include an OSRF stamp.